app/Filament/Server/Resources/UserResource/Pages/ListUsers.php

```diff
@@ -387,19 +387,17 @@ protected function getHeaderActions(): array
                 ->action(function (array $data, SubuserCreationService $service) use ($server) {
                     $email = strtolower($data['email']);
 
-                    if (in_array('console', $data['control'])) {
-                        $data['websocket'][0] = 'connect';
-                    }
-
                     $permissions = collect($data)->forget('email')->map(fn ($permissions, $key) => collect($permissions)->map(fn ($permission) => "$key.$permission"))->flatten()->all();
+                    $permissions = array_unique(array_merge($permissions, [Permission::ACTION_WEBSOCKET_CONNECT]));
 
                     try {
-                        $service->handle($server, $email, $permissions);
+                        $subuser = $service->handle($server, $email, $permissions);
 
                         Activity::event('server:subuser.create')
-                            ->subject($data['email'])
+                            ->subject($subuser->user)
```

Comment:
```suggestion
                            ->subject($subuser->user->email)
```
The raw user is stored in actor